wallet/wallet.c

```diff
@@ -414,7 +414,7 @@ static void db_set_utxo(struct db *db, const struct utxo *utxo)
 		assert(!utxo->reserved_til);
 
 	stmt = db_prepare_v2(
-		db, SQL("UPDATE outputs SET status=?, reserved_til=?"
+		db, SQL("UPDATE outputs SET status=?, reserved_til=? "
 			"WHERE prev_out_tx=? AND prev_out_index=?"));
```

Comment:
What about `" WHERE …"` instead of adding the space to end of previous line?

See lines [304](https://github.com/ElementsProject/lightning/blob/9b02c7827c8a4e864b8df1fa24f27325f18cf6ba/wallet/wallet.c#L304), 341, 518, 1219, etc.

Reply:
Both styles are used in this file. See lines: 75, 206, 214, 304, 553, 722, 734, 771, 1360, 1519, 1529, ... 
I can't decide whats better, since I don't see a difference. In general I would prefer to have the same style.
It would require a general decision and cleanup, which is not intended in this PR.

Reply:
It's pretty much personal preference, and we don't have a guideline. FWIW the same query would look like this if I were writing this in python:

```sql
UPDATE
  outputs
SET
  status=?,
  reserved_til=?
WHERE
  prev_out_tx=? AND
  prev_out_index=?
```

Which I find very readable and allows you to jump to any part of the query/statement you want. But multiline strings in C don't work.